source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,107 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
+
+#### Plugin Changes
+ - Added new frontend plugin extension point for the new messages separator bar.
+ - Added a new plugin extensibility point to add actions to the code blocks.
+ - Added the plugin hook ``UserHasBeenDeactivated``.
+ - Added a new server side plugin API method to set the searchable content for a file info (``SetFileSearchableContent``). The ``MessageHasBeenPosted`` plugin hook is now executed after the attachments are linked to the post.
+
+### Bug Fixes
+ - Fixed keyboard support for the left-hand side channel menu, the left-hand side category menu, and the post dot menu.
+ - Fixed display name in the ``comment_on`` component.
+ - Fixed an issue with keyboard support for some menus with submenus.
+ - Fixed an issue with a disappearing punctuation when following a group mention.
+ - Fixed an issue where compliance export jobs were not able to start after disabling and enabling the compliance export.
+ - Fixed a potential read after write issue when loading a license.
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Removed ``ServiceSettingsDefaultGfycatAPISecret`` and ``ServiceSettingsDefaultGfycatAPIKey`` configuration settings.
+ - Under ``TeamSettings`` in ``config.json``:
+    - Added a new config setting ``EnableJoinLeaveMessageByDefault`` that sets the default value for ``UserSetting``, ``ADVANCED_FILTER_JOIN_LEAVE``.
+ - Under ``DisplaySettings`` in ``config.json``:
+    - Added a setting ``MaxMarkdownNodes`` to limit the maximum complexity of markdown text on mobile.
+
+ #### Changes to Enterprise plan:
+ - Under ``DataRetentionSettings`` in ``config.json``:
+    - Added a new configuration setting ``RetentionIdsBatchSize``, which allows admins to to configure how many batches of IDs will be fetched at a time when deleting orphaned reactions. The default value is 100.
+
+### API Changes
+ - Added the ``X-Forwarded-For`` request header to the audit stream for all Rest API calls.
+ - Added API endpoint ``POST /api/v4/user/login/desktop_login``. Modified OAuth/SAML flows to include ``desktop_login`` where applicable.
+ - Added new API endpoint ``GET`` ``/api/v4/channels/<channel-id>/common_teams`` to fetch list of teams common between members of a group message.
+ - Added new API endpoint ``POST`` ``/api/v4/channels/<channel-id>/convert_to_channel`` to convert a group message to a private channel.
+ - Added a new ``MessageHasBeenDeleted`` hook to the plugin API.
+ - Moved the ``request`` package into the public shared folder.
+
+### Go Version
+ - v9.1 is built with Go ``v1.20.0``.
```

Comment:
1.20.7. Not sure from where you got the incorrect patch release version.

Reply:
Ah okay, I thought I had mentioned this, but the file to look for the exact version is `server/build/Dockerfile.buildenv`.